apps/web/src/components/NavMenusSection/data.tsx

```diff
@@ -0,0 +1,102 @@
+import type { ReactNode } from "react";
+import {
+  BookOpenIcon,
+  ChartBarIcon,
+  CommandLineIcon,
+  Squares2X2Icon,
+} from "@heroicons/react/24/solid";
+
+import { env } from "~/env.mjs";
+import EthereumIcon from "~/icons/ethereum.svg";
+import {
+  buildBlobStatsRoute,
+  buildBlockStatsRoute,
+  buildBlocksRoute,
+  buildTransactionStatsRoute,
+  buildTransactionsRoute,
+  buildBlobsRoute,
+} from "~/utils";
+
+function resolveApiUrl(): string {
+  if (env.NEXT_PUBLIC_NETWORK_NAME === "mainnet") {
+    return "https://api.blobscan.com";
+  }
+
+  return `https://api.${env.NEXT_PUBLIC_NETWORK_NAME}.blobscan.com`;
+}
+
+export type MenuItem = {
+  label: string;
+  href: string;
+  icon: ReactNode;
+  type: "single";
+};
+
+export type ExpandibleMenuItem = {
+  label: string;
+  icon: ReactNode;
+  items: {
+    label: string;
+    href: string;
+  }[];
+  type: "expandible";
+};
```

Comment:
Let's rename this types to `NavItem` and `ExpandibleNavItem`

Reply:
We're already using types similar to this one in the `MobileMenu` and  `NavItem`. Let's reuse these ones there

Reply:
Done c724c67 35e39a9 f9786ba